src/fprime_gds/flask/static/addons/commanding/command-input.js

```diff
@@ -83,7 +83,8 @@ Vue.component("command-input", {
         this.$root.$refs.command_input = this;
     },
     data: function() {
-        let selected = command_assignment_helper(null, [], "CMD_NO_OP");
+        // Select CMD_NO_OP by default if it exists, otherwise select the first command
+        let selected = command_assignment_helper("cmdDisp.CMD_NO_OP", [], "CMD_NO_OP");
```

Comment:
Without it, in the `command_assignment_helper()` function we match both `CMD_NO_OP` and `CMD_NO_OP_STRING`. 
Because commands are not ordered in the JSON dictionary, it happens that CMD_NO_OP_STRING appears first (at least in `Ref/`), and therefore the default command that is selected when the GDS opens is `CMD_NO_OP_STRING`. 

I wanted to keep `CMD_NO_OP` first, so instead of ordering the dictionary which sounded like overkill, I found this little trick.

The behavior is then
```
if `cmdDisp.CMD_NO_OP` exists, return
elif commands match `CMD_NO_OP`, return first match
else return first command in dictionary
```

Reply:
The drop-down _is_ sorted correctly, but that's because it sorts them when it retrieves them ([here](https://github.com/nasa/fprime-gds/blob/devel/src/fprime_gds/flask/static/addons/commanding/command-input.js#L211-L225)) from the datastore.
The commands in the datastore are not sorted because it comes straight from the dictionary, which is not sorted in the JSON implementation. So when retrieving the list of matches with `command_assignment_helper()`, it is not sorted correctly. Maybe it should be the responsibility of the JSON loader to sort the dictionaries?

Reply:
Is it worth sorting the dictionary at load time to offset this minor discrepancy (custom cmdDisp name and getting NO_OP_STRING instead of NO_OP)?  Otherwise I see no need to.

Reply:
I think we should fix this issue (either sorting at load time, or sorting before running this search) because:

1. Users will expect it to work consistently even with custom component names
2. It will take more work to document and explain the discrepancy than it will be to fix it
3. Confusion will cost us loyalty (even if it is an iota of loyalty) and as such we should document or fix.

Reply:
I have added a fix to sort the dictionaries at load time

Ideally they'd be sorted from the output of `fpp-to-dict`... but also we shouldn't be relying on it. I'm happy to keep it like this